.env

```diff
@@ -1,42 +1,63 @@
+# In all environments, the following files are loaded if they exist,
+# the latter taking precedence over the former:
+#
+#  * .env                contains default values for the environment variables needed by the app
+#  * .env.local          uncommitted file with local overrides
+#  * .env.$APP_ENV       committed environment-specific defaults
+#  * .env.$APP_ENV.local uncommitted environment-specific overrides
+#
+# Real environment variables win over .env files.
+#
+# DO NOT DEFINE PRODUCTION SECRETS IN THIS FILE NOR IN ANY OTHER COMMITTED FILES.
+#
+# Run "composer dump-env prod" to compile .env files for production use (requires symfony/flex >=1.2).
+# https://symfony.com/doc/current/best_practices.html#use-environment-variables-for-infrastructure-configuration
+
 ###> symfony/framework-bundle ###
 APP_ENV=dev
 APP_DEBUG=1
-APP_SECRET=73fd814e2c9fbf61dc5f87a62967d829
-#TRUSTED_PROXIES=127.0.0.1,127.0.0.2
-#TRUSTED_HOSTS='^localhost|example\.com$'
+APP_SECRET=!ChangeMe!
+TRUSTED_PROXIES=127.0.0.0/8,10.0.0.0/8,172.16.0.0/12,192.168.0.0/16
```

Comment:
This is linked to the docker configuration, is it necessary to take it back to the `bolt/core` repository?`10.0.0.0/8,172.16.0.0/12,192.168.0.0/16`

Reply:
I copied this from `bolt/project`, figuring it'd make sense to keep them the same. What do you think this line should say?

Reply:
I just corrected that in this [pull request](https://github.com/bolt/project/pull/47/files)